app/test_main.py

```diff
@@ -1,3 +1,48 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_year, dog_year, expect_error", [
+        pytest.param("5", 6.1, TypeError, id="cat value is not integer"),
+        pytest.param(5, "6", TypeError, id="dog value is not integer")
+    ]
+)
+def test_cheking_exeptions(cat_year: int,
+                           dog_year: int,
+                           expect_error: Exception) -> None:
```

Comment:
Here must be `Type[Exception]` to signify that you expect an exception as a class not an instance